composer.json

```diff
@@ -46,14 +46,14 @@
         ],
         "post-install-cmd": [
             "php artisan clear-compiled",
-            "php artisan optimize",
-            "bash bin/setup.sh"
+            "php artisan optimize"
         ],
         "pre-update-cmd": [
             "php artisan clear-compiled"
         ],
         "post-update-cmd": [
-            "php artisan optimize"
+            "php artisan optimize",
+            "bash bin/setup.sh"
```

Comment:
copy line 56 between lines 52 and 53 - and it's fixed re my original report.

Keep the line for executing command within vagrant - that should work although no need for semicolon, you could have directly enter the path to `phpunit`